engines/bastion/app/assets/javascripts/bastion/components/bst-edit.directive.js

```diff
@@ -335,6 +335,74 @@ angular.module('Bastion.components')
             unbindWatcher();
         });
     }])
+    .directive('bstCheckboxList', function () {
```

Comment:
What's this new component do for us?

Reply:
I'm honestly not sure if a new component is the way to go here.  I adapted it from the existing `bst-edit-multiselect` so that I could eliminate the "Save" and "Cancel" buttons.

Reply:
also eliminated the "X" button that was active while editing, and eliminated the ability to exit edit mode